components/widgets/inputs/channels_input.test.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {shallow} from 'enzyme';
+import React from 'react';
+
+import {Channel} from 'mattermost-redux/types/channels';
+
+import ChannelsInput from './channels_input';
+
+describe('components/widgets/inputs/ChannelsInput', () => {
+    test('should match snapshot', () => {
+        const wrapper = shallow(
+            <ChannelsInput
+                placeholder='test'
+                ariaLabel='test'
+                onChange={jest.fn()}
+                channelsLoader={jest.fn()}
+                onInputChange={jest.fn()}
+                inputValue=''
+                value={[
+                    {
+                        id: 'test-channel-1',
+                        type: 'O',
+                        display_name: 'test channel 1',
+                    } as Channel,
+                    {
+                        id: 'test-channel-2',
+                        type: 'P',
+                        display_name: 'test channel 2',
+                    } as Channel,
+                ]}
+            />,
+        );
+        expect(wrapper).toMatchInlineSnapshot(`
```

Comment:
just curious why do we need to use inline snapshots here for this test?

Reply:
This was a js -> ts file conversion that got counted as an add and delete. Not sure why the original author made it inline versus its own file. In general I don't like snapshots because I think they do a really poor job capturing what you're asserting on.